tasks/html-css-popup.md

```diff
@@ -25,6 +25,43 @@ The goal of this task is to practice coding HTML & CSS from mockup. You are expe
     makes the content of the popup scrollable
 - mobile view is irrelevant for this task
 
+## Recommendations
+- Make sure that your navigation is built by this structure: 
+``` HTML
+  <nav>
+    <ul>
+      <li>
+        <a href='#'>Link</a>
+      </li>
+    <ul>
+  </nav>
+```
+- Make sure that you've used visually-hidden for hiding inputs:
+``` CSS
+.visually-hidden {
```

Comment:
Same here. Please, educate students why this is recommended.

Reply:
I've updated the description a little bit here. Also, I added the video where students can watch more about this technique